bin/build_fixups.sh

```diff
@@ -14,6 +14,12 @@ thisdir=`dirname $realpath`
 if [ -d $AOMP/examples ] ; then 
   $SUDO rm -rf $AOMP/examples
 fi
+
+if [[ "$DEVEL_PACKAGE" =~ "devel" ]]; then
+  AOMP="$AOMP/""$DEVEL_PACKAGE/share/openmp"
+  echo "Request for devel package found."
+  mkdir -p $AOMP/share/openmp
+fi
```

Comment:
This is not needed due to being handled in the build_openmp_extras.sh script.

Reply:
Removed the snippet